data-model/SDP-data-model-templates/manuscript-object_sdp-template.json

```diff
@@ -39,35 +39,124 @@
     {
       "id": "URI",
       "type": "CV",
-      "note": [
-        "array of strings"
-      ],
-      "associated_dates": [
+      "associated_date": [
         {
+          "date_type": "CV",
           "date_human_readable": "",
           "date_normalized": "ISO-8601 Dates",
           "note": [
             "array of strings"
           ]
         }
       ],
-      "associated_entities": [
+      "associated_person": [
+        {
+          "relation": "CV",
+          "id": "URI",
+          "note": [
+            "array of strings"
+          ]
+        }
+      ],
+      "associated_place": [
         {
-          "entity_type": "CV",
-          "role": "CV",
-          "id": "URI/ARK",
+          "relation": "CV",
+          "id": "URI",
           "note": [
             "array of strings"
           ]
         }
       ],
+      "note": [
+        "array of strings"
+      ],
       "sources": [
-        "ARK/ID of a Paratext? URI of a Zotero Bibl?"
+        "Paratext ARKs | Zotero URI(?)"
       ]
     }
   ],
-  "associated_entities": {
-    "associated_person": [
+  "paratexts": {
```

Comment:
I keep going back and forth on what to do about manuscript object paratexts. I think there's a distinction between the 'scope' of a paratext (what object type it's 'about') and the location of the paratext. I'm still thinking through ways to capture different object scopes, maybe within the taxonomy of paratext types or maybe through additional fields. 

For location, though, I wonder if paratexts in pastedowns or even fly leaves would be entered at the manuscript object level?